packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -220,7 +222,7 @@ const ReleasesPage = () => {
     setReleaseModalShown((prev) => !prev);
   };
 
-  if (isLoading) {
+  if (isLoading || isLoadingSettings) {
```

Comment:
I recommend renaming isLoading to something more specific now that there are several things loading